### ISSUE TITLE
Definition of RTI

### ISSUE THREAD

**Issue by JanNehyba** [andrewmcaleavey/ReliableTrendIndex#15]
Dear Andrew @andrewmcaleavey, 

I discovered your pre-print on RTI (When (Not) to Rely on the Reliable Change Index). I am excited about it and have tried installing your R package. 
However, I have a relatively fundamental question for you, which will probably reveal my ignorance in the field of statistics, but I wanted to ask how is RTI defined? I couldn't find it anywhere, only when running ?reliableTrend(). I found that the "Reliable Trend Index is the "z-value of a linear trend considering a fixed measurement error at each observation." If I understand correctly, RTI is the z-value of a linear trend calculated using Deming regression with orthogonal deviations? RTI = |(b/SE(b))|; where b is the slope of the linear regression, and SE(b) is the standard error of the slope of the linear regression.
I would like to use your approach to idiographic methods in collecting feedback for teacher from pupils in secondary school. 

Thank you very much for the clarification, Jan 
https://www.muni.cz/en/people/106930-jan-nehyba/publications

PS:  I got an error (This form isn't set up yet ) on your contact form on your site, so I am writing here.

**Comment by andrewmcaleavey** [comment]
Hi Jan, 

Thanks for writing. I'll make a more clearly-defined formula and place it in the documentation!

I think your formula is essentially correct, assuming that the regression is computed with a known error variance in the appropriate variable. 

My understanding of Deming regression is limited, but doesn't an orthogonal approach assume that both variables are observed with equal errors? Can Deming regression incorporate models with error in only one variable? In the case that the RTI was intended, one variable is usually time and observed essentially without error, while the other is a self-report scale with considerable error. 

Either way, it is certainly a relevant parallel model I hadn't thought of including in the documentation. If it includes the cases with just one error term, it would likely be the same as the underlying model for the RTI (and I would want to make that explicit).

**Comment by JanNehyba** [comment]
You are correct that the Deming regression assume that both variables are observed with equal errors (errors in both variables are correlated). I'm not a mathematician or statistician; instead, I read it from various notes. So is a regression in RTI closer to the OLS regression method, which assumes the presence of errors only in the dependent variable? Probably OLS regression is a special use case of the Errors-in-variables regression.

**Comment by andrewmcaleavey** [comment]
I significantly changed the backend and definition lately, this issue is better addressed now. The documentation provides a definition and closed-form formula for fixed time intervals. Closing the issue for now.
